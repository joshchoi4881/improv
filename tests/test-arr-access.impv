func int main() {
  int[] int_arr;
  note[] note_arr;

  int_arr = [1, 2, 3, 4, 5];
  printi(int_arr[1]);

  note_arr = [<1, "wh">, <5, "wh">];
  printn(note_arr[1]);
  
  return 0;
}
