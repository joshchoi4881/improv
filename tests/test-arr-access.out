2
<5, "wh">